### ISSUE TITLE
train XG boost tree on XBT data

### ISSUE THREAD

**Issue by stevehadd** [Fracappo87/XBTs_classification#43]
XG boost represents the current state of the art in decision tree ensembles. It has a scikit-learn like interface, so should be easy to integrate into the current workflow. tutorial:
https://machinelearningmastery.com/develop-first-xgboost-model-python-scikit-learn/

**Comment by stevehadd** [comment]
we will need to do some hyperparameter tuning to get the best results:
https://www.kaggle.com/prashant111/a-guide-on-xgboost-hyperparameters-tuning
https://blog.cambridgespark.com/hyperparameter-tuning-in-xgboost-4ff9100a3b2f

**Comment by stevehadd** [comment]
With the refactoring required to accomodate gradient boosted trees, we should rerun all the classification notebooks and experiment definitions to check that they still work.
